README.md

```diff
@@ -73,22 +74,22 @@ If `err` is passed, no operation should be made in `conn`.
 ### Dial(open/create) a new stream
 
 
-- `Node.js` stream = muxedConn.newStream([function (err, stream)])
+- `JavaScript` stream = muxedConn.newStream([function (err, stream)])
 - `Go` stream, err := muxedConn.newStream()
 
 This method negotiates and opens a new stream with the other endpoint.
 
 If `err` is passed, no operation should be made in `stream`.
 
-`stream` interface our established Stream with the other endpoint, it must implement the [Duplex Stream interface](https://nodejs.org/api/stream.html#stream_class_stream_duplex) in Node.js or the [ReadWriteCloser](http://golang.org/pkg/io/#ReadWriteCloser) in Go.
+`stream` interface our established Stream with the other endpoint, it must implement the [Duplex pull-stream interface](https://pull-stream.github.io) in JavaScript or the [ReadWriteCloser](http://golang.org/pkg/io/#ReadWriteCloser) in Go.
 
-In the Node.js case, if no callback is passed, stream will emit an 'ready' event when it is prepared or a 'error' event if it fails to establish the connection, until then, it will buffer the 'write' calls.
+In the JavaScript case, if no callback is passed, stream will emit an 'ready' event when it is prepared or a 'error' event if it fails to establish the connection, until then, it will buffer the 'write' calls.
```

Comment:
I don't believe this holds true anymore, maybe it is not even needed. @dignifiedquire